modin/_compat/pandas_api/py36/base.py

```diff
@@ -29,8 +29,8 @@ class Python36CompatibleBasePandasDataset(BaseCompatibleBasePandasDataset):
     """Compatibility layer for 'Python 3.6 pandas' for Dataset."""
 
     def _validate_ascending(self, ascending):  # noqa: PR01, RT01
```

Comment:
Do we need this method?

Reply:
this is for uniform API between different compatibility classes, we'll likely add more in the future...

Reply:
I see, thanks!